src/client/components/HealthCareApp.jsx

```diff
@@ -173,19 +179,36 @@ class HealthCareApp extends React.Component {
         </div>
       );
     }
+    let devPanel = undefined;
+    if (__DEV__) {
+      const queryParams = _.fromPairs(
+        window.location.search.substring(1).split('&').map((v) => { return v.split('='); }));
```

Comment:
I have never done this, but it looks like react router provides the query params:
Here is an example from here : https://github.com/reactjs/react-router/blob/8707d05b00273590e4d054771b8ae7ae9ad312e3/examples/query-params/app.js
    let { query } = this.props.location
    let age = query && query.showAge ? '33' : ''